server/app/views/questiontypes/NameQuestionFragment.html

```diff
@@ -11,7 +11,7 @@
   ></div>
 
   <!-- TODO: errors -->
-  <div th:with="firstNameId=${#strings.randomAlphanumeric(8)}">
+  <div th:with="firstNameId=${#strings.randomAlphanumeric(8)}" class="cf-name-first">
```

Comment:
these changes appear unrelated, but the enumerator test looks up the name question inputs using these classes, so added them here